types/file-saver-es/test/file-saver-es-tests.ts

```diff
@@ -28,13 +28,13 @@ function testDeprecatedSaveAs() {
  * @summary Test for "saveAs" function on the window object.
  */
 function testWindowSaveAs() {
```

Comment:
Like above, I don't think testing `window` is needed.

Reply:
As I said, nothing I can do about it, the library exposes stuff, type definitions only describe what the library does.